README.md

```diff
@@ -121,6 +121,86 @@ steps:
           pull-request-metadata-key: "github-pull-request-plugin-number"
 ```
 
+## Example Pipeline
+
+This is an example pipeline which ties everything together to produce nicely
+annotated bundle update pull requests.
+
+This pipeline requires two secrets:
+
+* Write access to the project GIT repository, by way of an SSH Key. This write
+  access is used for pushing up the bundle update commit.
+
+* Github API access, by populating the environment variable `GITHUB_TOKEN` with
+  a personal access token providing `repo` access to the repository. This is
+  used for opening the pull request and adding comments.
+
+It's recommended to use the [AWS S3 Secrets Buildkite Plugin] to provide these
```

Comment:
> [AWS S3 Secrets Buildkite Plugin]
Is this mean to reference that plugin?

Reply:
Yeah, the link is at the bottom of the document. Although when I try it, it links a little deep. 

It now links to the uploading secrets section (7bfbf71).

Reply:
Sorry missed that, thanks!